src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/hooks/useFeatureEnabled.tsx

```diff
@@ -5,22 +5,27 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
+import { useMemo } from 'react'
 import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
 import { useFeature } from '@common/hooks/useFeatures'
 import { usePermission } from '@rbac/hooks/usePermission'
 import { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
 import { ResourceType } from '@rbac/interfaces/ResourceType'
 import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
 
-interface UseFeatureEnabledReturn {
+export interface UseFeatureEnabledReturn {
   enabledByPermission: boolean
   enabledByPlanEnforcement: boolean
   featureEnabled: boolean
   canEdit: boolean
   canToggle: boolean
 }
 
-const useFeatureEnabled = (): UseFeatureEnabledReturn => {
+export interface UseFeatureEnabledProps {
+  tags?: string[]
+}
+
+const useFeatureEnabled = ({ tags = undefined }: UseFeatureEnabledProps): UseFeatureEnabledReturn => {
```

Comment:
Since it's optional, it should be `undefined` by default.
```suggestion
const useFeatureEnabled = ({ tags }: UseFeatureEnabledProps): UseFeatureEnabledReturn => {
```

Reply:
Would it make sense to default this to a blank array so you can rely on it being an array and mean you can kill off optional chaining?